eng/install-test-runner-prerequisites.ps1

```diff
@@ -16,9 +16,9 @@ function Install-PowerShellTools {
 
 
 function Update-Path {
-    $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") +
-                ";" +
-                [System.Environment]::GetEnvironmentVariable("Path","User")
+    $env:Path = [System.Environment]::GetEnvironmentVariable("Path", "Machine") +
+    ";" +
+    [System.Environment]::GetEnvironmentVariable("Path", "User")
```

Comment:
I like the extra space after the commas, but would prefer to restore the extra indentation on lines 20 and 21.